code/game/machinery/portable_turret.dm

```diff
@@ -902,7 +902,7 @@ Status: []<BR>"},
 
 /obj/machinery/porta_turret_cover/attack_hand(mob/user)
 	. = ..()
-	if(.)
+	if(. || !Parent_Turret)

```

Comment:
can't accept it if you can't reproduce it and aren't aware of what's the bug, the error could be something else